### DISCUSSION TITLE
Multiple docs / Monorepo support

### DISCUSSION THREAD

**Discussion by akil-rails** [vuejs/vitepress#1160] [Q&A]
I've to document multiple _products_  with shared components, and branding. The layouts can be different in the future (though they are the same today). What's the recommended way to do this ?

- [ ] create multiple vitepress projects 
- [ ] anything else?

**Comment by emersonbottero** [comment]
I do that at work actually, can't share the repo though.. 
I have a single repo for docs and a pipeline that clones all repos from my group (using github api)
after that I move all md, image, and relevant files to the docs folders (In my case I use repo Description to group repos together, like Global Tools
then I build the hole thing and the docs from all repos are placed in on single doc.
the only problems I have so far are:
- edit link wont work
- if one repo has broken links the doc wont build
